federation/pkg/kubefed/util/util.go

```diff
@@ -210,3 +210,26 @@ func WaitForPods(clientset *client.Clientset, pods []string, namespace string) e
 	})
 	return err
 }
+
+func GetFirstNodeIP(clientset *client.Clientset) ([]string, error) {
```

Comment:
oops, sorry i should have checked. at first it seemed simply straight forward. I should be reusing this function [GetPreferredNodeAddress](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/node/node.go#L57). Thanks for pointing this issue.